RobotFramework_Testsuites/Config/CConfig.py

```diff
@@ -255,7 +255,7 @@ def loadCfg(self):
             ROBFW_AIO_Data.update({key:v})
         oJsonPreprocessor = CJsonPreprocessor(syntax="python", currentCfg=ROBFW_AIO_Data)
         try:
-            oJsonCfgData = oJsonPreprocessor.jsonLoad(self.sTestCfgFile)
+            oJsonCfgData = oJsonPreprocessor.jsonLoad(self.sCalcAbsPath(self, self.sTestCfgFile))
```

Comment:
Hi Son,
static method should be called with CConfig instead of self.
Thank you,
 Thomas